tests/cypress/support/commands.js

```diff
@@ -146,38 +146,38 @@ Cypress.Commands.add('openTaskJob', (taskName, jobID = 0, removeAnnotations = tr
     cy.openJob(jobID, removeAnnotations);
 });
 
+Cypress.Commands.add('interactControlButton', (objectType) => {
+    cy.get('body').click('top');
```

Comment:
Is clicking on exactly 'top' important?

Reply:
click() without specifying coordinates produces mouse events in the center of the object. In this case, it turns out that he clicks on the canvas. There may also be a drawn object under the cursor. Which can lead to unexpected consequences.

Reply:
Did you try using focus() instead of clicking?

Reply:
I'll double-check. Maybe it would be better.

Reply:
Changed to focus().